test/Icons.Test/Services/IconFetchingServiceTests.cs

```diff
@@ -13,7 +13,7 @@ public class IconFetchingServiceTests : ServiceTestBase<IconFetchingService>
     [InlineData("hopin.com")] // uses svg+xml format
     [InlineData("ameritrade.com")] // redirects to tdameritrace.com
     [InlineData("icloud.com")]
-    [InlineData("bofa.com", Skip = "Broken in pipeline for .NET 6. Tracking link: https://bitwarden.atlassian.net/browse/PS-982")]
+    [InlineData("bofa.com")]
```

Comment:
@MGibson1 I restored the this test given the comment here but do you have any more info on it? I thought your rewrite of the service might resolve it.